app/components/ZustandModal.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import useModalZustandStore from '../hooks/useModalZustandStore';
 
-const Modal = () => {
+const ZustandModal = () => {
```

Comment:
changed `Modal` to `ZustandModal` to distinguish it from others (such as RecoilModal)